Python/instrumentation.c

```diff
@@ -642,10 +633,11 @@ de_instrument_per_instruction(PyCodeObject *code, int i)
     int original_opcode = code->_co_monitoring->per_instruction_opcodes[i];
     CHECK(original_opcode != 0);
     CHECK(original_opcode == _PyOpcode_Deopt[original_opcode]);
-    instr->op.code = original_opcode;
+    *opcode_ptr = original_opcode;
     if (_PyOpcode_Caches[original_opcode]) {
         instr[1].cache = adaptive_counter_warmup();
```

Comment:
The cache would be initialized here even if the instr is `INSTRUMENTED_LINE` (but the `original_opcode` is an instruction with cache). I guess that's no harm?

Reply:
This is indeed harmless.